src/test/kfs_rest_test.cpp

```diff
@@ -63,9 +63,9 @@ class MockedServer : public Server {
 
     std::unique_ptr<Module> createModule(const std::string& name) override {
         if (name == ovms::SERVABLE_MANAGER_MODULE_NAME)
-            return std::make_unique<MockedServableManagerModule>();
+            return std::make_unique<ovms::ServableManagerModule>();
         if (name == ovms::GRPC_SERVER_MODULE_NAME)
-            return std::make_unique<MockedGRPCServerModule>(*this);
+            return std::make_unique<ovms::GRPCServerModule>(*this);
```

Comment:
If we don't use those Mocks then we can delete this class as well and start ovms::Server? Or are there different impediments?